web/templates/settings/account_settings.hbs

```diff
@@ -68,7 +68,7 @@
                 {{/if}}
             </form>
 
-            <div class="input-group">
```

Comment:
Removing this class decreases the space between buttons and "Privacy" heading which is not intended for this PR. We can keep both the classes.